airbyte/_util/api_util.py

```
@@ -858,3 +859,102 @@ def check_connector(
+) -> None:
+    """Override the docker image and tag for a specific connector.
+
+    https://github.com/airbytehq/airbyte-platform-internal/blob/10bb92e1745a282e785eedfcbed1ba72654c4e4e/oss/airbyte-api/server-api/src/main/openapi/config.yaml#L5111
+    """
+    _ = workspace_id
+    json_result = _make_config_api_request(
+        path="/scoped_configuration/create",
+        json={
+            # https://github.com/airbytehq/airbyte-platform-internal/blob/10bb92e1745a282e785eedfcbed1ba72654c4e4e/oss/airbyte-api/server-api/src/main/openapi/config.yaml#L7376
+            "value": override.docker_image_override,
+            "config_key": "docker_image",  # TODO: Fix this.
```

Ruff (0.8.2)

Missing issue link on the line following this TODO

(TD003)